pluginconfig/commit.go

```diff
@@ -29,11 +29,14 @@ const (
 	defaultInflightPriceCheckRetries          = 5
 	defaultAsyncObserverSyncFreq              = 5 * time.Second
 	defaultAsyncObserverSyncTimeout           = 10 * time.Second
+	defaultExecNoDeviationThresholdUSDWei     = 10e9
+	defaultDataAvNoDeviationThresholdUSDWei   = 20e9
 )
 
 type FeeInfo struct {
-	ExecDeviationPPB             cciptypes.BigInt `json:"execDeviationPPB"`
-	DataAvailabilityDeviationPPB cciptypes.BigInt `json:"dataAvailabilityDeviationPPB"`
+	ExecDeviationPPB              cciptypes.BigInt `json:"execDeviationPPB"`
+	DataAvailabilityDeviationPPB  cciptypes.BigInt `json:"dataAvailabilityDeviationPPB"`
+	DisableChainFeeDeviationCheck bool             `json:"disableChainFeeDeviationCheck"`
```

Comment:
For consistency's sake
```suggestion
	ChainFeeDeviationDisabled bool             `json:"disableChainFeeDeviationCheck"`
```